content/en/xtc-release-notes/v40.1.md

```diff
@@ -0,0 +1,26 @@
+---
+title: XTC v40.1
+linkTitle: XTC v40.1
+
+weight: 177
+type: docs
+
+date: 2021-10-13
+
+description: >
+---
+
+## XTC v40.1
+
+
+
+### Load Testing
+- Fixed an issue when displaying load test reports. For reports that have been created months ago, the report pages show binary content instead of proper HTML.
+
+#### Resource Usage Tracking
```

Comment:
Also, everything in this page below this line does not belong in v40.1 (check document).

Reply:
really not sure. Why I did that.